src/theme.jsx

```diff
@@ -53,9 +54,17 @@ const theme = createMuiTheme({
       root: {
         height: 24,
         marginRight: 1,
-        backgroundColor: SECONDARY.DARK,
+        backgroundColor: PRIMARY.MAIN,
         color: 'white',
       },
+      clickable: {
+        '&:focus': {
+          backgroundColor: PRIMARY.MAIN,
+        },
+        '&:hover': {
+          backgroundColor: emphasize(SECONDARY.DARK),
+        },
+      },
```

Comment:
Since we're using a LinkIcon with hover on the summary column, we can remove the hover that is applied on the full row. This will also help with the issue of being navigated away from the site when clicking outside the chip area by accident.

Remove `MuiTableRow`.